dir.props

```diff
@@ -103,7 +103,12 @@
     <NoExplicitReferenceToStdLib>true</NoExplicitReferenceToStdLib>
     <AddAdditionalExplicitAssemblyReferences>false</AddAdditionalExplicitAssemblyReferences>
     <GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>
-    <LangVersion>latest</LangVersion>
+    <LangVersion>8.0</LangVersion>
+    <UseSharedCompilation>true</UseSharedCompilation>
+    <BuildingNETCoreAppVertical>true</BuildingNETCoreAppVertical>
+    <IncludeVSTestReferences>false</IncludeVSTestReferences>
+    <CopyConfigurationFiles>false</CopyConfigurationFiles>
+    <RuntimeConfigPath>$(PackagesDir)/xunit.runner.console/$(XUnitPackageVersion)/tools/netcoreapp2.0/xunit.console.runtimeconfig.json</RuntimeConfigPath>
```

Comment:
These four (`BuildingNETCoreAppVertical`, `IncludeVSTestReferences`, `CopyConfigurationFiles`, and `RuntimeConfigPath`) are working around the `test.targets` from BuildTools.

It ensures that:
* `RunTests.cmd` is generated to use `dotnet.exe xunit.console.dll`
* `xunit.console.runtimeconfig.json` stays targeting netcoreapp2.0 (rather than v9.9.9)

Reply:
Yeah sorry for all that stuff which isn't really needed outside of corefx. I guess at some point corert should either migrate away from tests.targets or clean it up...